packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -8,6 +8,54 @@
 />
 
 <style>
+  @font-face {
+    font-family: "Aeonik";
+    src: url("https://tc2static.oktacdn.com/assets/courage/font/assets/Aeonik-Regular.4957ae92e5585e501e94.woff")
+      format("woff");
+    font-weight: normal;
+    font-style: normal;
+  }
+
+  @font-face {
+    font-family: "Aeonik";
+    src: url("https://tc2static.oktacdn.com/assets/courage/font/assets/Aeonik-Medium.3b09f560a7da8d6f25b5.woff")
```

Comment:
@KevinGhadyani-Okta Some of these are the same URL. Is that correct? Also, are these public? May not legally be able to check these URLs into Github.